tests/ui/coroutine/reborrow-mut-upvar.rs

```diff
@@ -1,9 +1,9 @@
-// run-pass
+// build-pass
 
 #![feature(coroutines)]
 
 fn _run(bar: &mut i32) {
-    || { //~ WARN unused coroutine that must be used
```

Comment:
Maybe it makes sense to add `#[warn(...)]` attributes to keep the warnings when they are significant.